apimanager/metrics/views.py

```diff
@@ -343,7 +345,7 @@ def get_total_number_of_apps(self, cleaned_data, from_date, to_date):
         apps_list =  self.get_all_consumers()
 
         for app in apps_list:
-            app_created_date = datetime.datetime.strptime(app["created"], API_DATETIMEFORMAT)
+            app_created_date = datetime.datetime.strptime(app["created"], '%Y-%m-%dT%H:%M:%SZ')
```

Comment:
@Reena-cell What is the difference here? See pimanager/apimanager/settings.py:247:23?